diff --git a/PauseMenu.rb b/PauseMenu.rb
index ae5707d..830a9f9 100644
--- a/PauseMenu.rb
+++ b/PauseMenu.rb
@@ -140,7 +140,7 @@ class PokemonMenu
     end
     commands[cmdOption=commands.length]=_INTL("Options")
     commands[cmdControls=commands.length]=_INTL("Controls")    
-    commands[cmdDebug=commands.length]=_INTL("Debug") if $DEBUG
+    commands[cmdDebug=commands.length]=_INTL("Debug")
     commands[cmdEndGame=commands.length]=_INTL("Quit Game")
     $game_screen.getTimeCurrent(true)
     loop do
