diff --git a/Battle_Move.rb b/Battle_Move.rb
index bb5ad81..ff7eb03 100644
--- a/Battle_Move.rb
+++ b/Battle_Move.rb
@@ -1280,8 +1280,14 @@ class PokeBattle_Move
     stagediv=[8,7,6,5,4,3,2,2,2,2,2,2,2]
     type=pbType(attacker)
     ##### Calcuate base power of move #####
-   
+
     basemult=1.0
+    if attacker.crested && attacker.species = :WHIMSICOTT && (PBStuff::BULLETMOVE).include?(@move)
+      basemult*=1.2
+    elsif attacker.crested && attacker.species = :INTELEON
+      basemult+=0.4*@priority
+    end
+
     #classic prep stuff
     attitemworks = attacker.itemWorks?(true)
     oppitemworks = opponent.itemWorks?(true)
diff --git a/Battler.rb b/Battler.rb
index d1a2cab..02041e8 100644
--- a/Battler.rb
+++ b/Battler.rb
@@ -5569,6 +5569,16 @@ class PokeBattle_Battler
           user.effects[:Outrage]=0
         end
       end
+
+      if user.species == :VICTINI && user.crested && basemove.move == :VCREATE && ((@battle.pbCanChooseNonActive?(self.index) && !@battle.pbAllFainted?(@battle.pbParty(self.index))) || @battle.pbIsWild?)
+        if @battle.pbIsWild?
+            return if @battle.cantescape || $game_switches[:Never_Escape]
+            @battle.decision=3 # Set decision to escaped
+        else
+          self.userSwitch = true
+        end
+      end
+
       # Switch moves
       for i in @battle.battlers
         if i.userSwitch
diff --git a/DataObjects - Yeeters.rb b/DataObjects - Yeeters.rb
index 9a64652..c52e2ff 100644
--- a/DataObjects - Yeeters.rb	
+++ b/DataObjects - Yeeters.rb	
@@ -149,6 +149,7 @@ def itemDump
     # crest = [864]
     # or
     # crest = (864..865).to_a
+    crest = (864..873).to_a
   end
   exporttext = "ITEMHASH = {\n"
   for item in $cache.items
@@ -287,6 +288,7 @@ def itemDumpCurrent
     # crest = [864]
     # or
     # crest = (864..865).to_a
+    crest = (864..873).to_a
     exclusions = [49]
     memory = (655..670).to_a
     mint = (781..801).to_a
diff --git a/Evolution.rb b/Evolution.rb
index 45b1509..2828ad0 100644
--- a/Evolution.rb
+++ b/Evolution.rb
@@ -825,6 +825,7 @@ end
 def checkEvolutionEx(pokemon)
   return nil if pokemon.species.nil? || pokemon.isEgg?
   return nil if [:EVERSTONE,:EEVIUMZ,:EVIOLITE].include?(pokemon.item)
+  return nil if pokemon.item == :VIGCREST && pokemon.species == :VIGOROTH
   ret=nil
   d= pbGetEvolvedFormData(pokemon.species,pokemon)
   if !d.nil?
diff --git a/PBStuff.rb b/PBStuff.rb
index ddb8060..d6f1c60 100644
--- a/PBStuff.rb
+++ b/PBStuff.rb
@@ -460,6 +460,11 @@ POKEMONTOCREST = {
   :BASTIODON => :BASTCREST,
   # Add Pokemon to custom Crests here.
   # :SOMEPOKEMON => :CUSTOMCREST,
+  :ARCHEOPS => :ARCHCREST,            :STARMIE => :STARMCREST,
+  :VICTINI => :VICREST,               :JUMPLUFF => :JUMPCREST,
+  :CHATOT => :CHATCREST,              :MARACTUS => :MARACREST,
+  :VIGOROTH => :VIGCREST,             :WHIMSICOTT => :WHIMSICREST,
+  :GENGAR => :GENGCREST,              :INTELEON => :INTELECREST,
 }
 
 SILVALLYCRESTABILITIES = {
diff --git a/Pokemon Desolation/DesoPokeBattle_Battle.rb b/Pokemon Desolation/DesoPokeBattle_Battle.rb
index 119adf7..f6d8f21 100644
--- a/Pokemon Desolation/DesoPokeBattle_Battle.rb	
+++ b/Pokemon Desolation/DesoPokeBattle_Battle.rb	
@@ -7,6 +7,23 @@ class PokeBattle_Battle
   def pbCrestEffects(index,pokemon)
     return if !@battlers[index].crested
     case @battlers[index].species
+    when :ARCHEOPS
+      if @battlers[index].pbCanReduceStatStage?(PBStats::ATTACK)
+        @battlers[index].pbReduceStat(PBStats::ATTACK,1,statdropper: @battlers[index].pbOpposing1)
+      end
+    when :JUMPLUFF
+      if !@battlers[index].pbOpposing1.isFainted?
+        opposing=@battlers[index].pbOpposing1
+        if opposing.pbCanReduceStatStage?(PBStats::SPDEF)
+          opposing.pbReduceStat(PBStats::SPDEF,1,statdropper: @battlers[index])
+        end
+      end
+      if !@battlers[index].pbOpposing2.isFainted?
+        opposing=@battlers[index].pbOpposing2
+        if opposing.pbCanReduceStatStage?(PBStats::SPDEF)
+          opposing.pbReduceStat(PBStats::SPDEF,1,statdropper: @battlers[index])
+        end
+      end
     end
   end
 end
diff --git a/Pokemon Desolation/DesoPokeBattle_Battler.rb b/Pokemon Desolation/DesoPokeBattle_Battler.rb
index 0586e6e..1d4fc78 100644
--- a/Pokemon Desolation/DesoPokeBattle_Battler.rb	
+++ b/Pokemon Desolation/DesoPokeBattle_Battler.rb	
@@ -15,6 +15,20 @@ class PokeBattle_Battler
 
   def crestStats
     case @crested
+    when :GENGAR
+      @ability = :MISTYSURGE
+      @type2 = :FAIRY
+    when :STARMIE
+      @ability = :PSYCHICSURGE
+    when :ARCHEOPS
+      @ability = :DEFIANT
+    when :CHATOT
+      @ability = :PUNKROCK
+    when :MARACTUS
+      @ability = :SANDRUSH
+    when :VIGOROTH
+      @ability = :GORILLATACTICS
+      @spdef, @defense = @spdef * 1.5, @defense * 1.5
     end
   end
 end
diff --git a/Pokemon Desolation/itemtext.rb b/Pokemon Desolation/itemtext.rb
index e9bd67c..e1fd911 100644
--- a/Pokemon Desolation/itemtext.rb	
+++ b/Pokemon Desolation/itemtext.rb	
@@ -7761,4 +7761,94 @@ ITEMHASH = {
 # 	:noUseInBattle => true,
 # 	:noUse => true,
 # },
+:STARMCREST => {
+	:ID => 864,
+	:name => "Starmie Crest",
+	:desc => "Grants Psychic Surge.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:ARCHCREST => {
+	:ID => 865,
+	:name => "Archeops Crest",
+	:desc => "Grants Defiant. Upon switching in, opponent Intimidates.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:VICREST => {
+	:ID => 866,
+	:name => "Victini Crest",
+	:desc => "V-create acts like U-turn.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:JUMPCREST => {
+	:ID => 867,
+	:name => "Jumpluff Crest",
+	:desc => "Upon switch in, decrease opponents' SpDef by 1 stage.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:CHATCREST => {
+	:ID => 868,
+	:name => "Chatot Crest",
+	:desc => "Grants Punk Rock.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:MARACREST => {
+	:ID => 869,
+	:name => "Maractus Crest",
+	:desc => "Grants Sand Rush.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:VIGCREST => {
+	:ID => 870,
+	:name => "Vigoroth Crest",
+	:desc => "Grants Gorilla Tactics, Eviolite, and Everstone.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:WHIMSICREST => {
+	:ID => 871,
+	:name => "Whimsicott Crest",
+	:desc => "Grants Bombardier.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:GENGCREST => {
+	:ID => 872,
+	:name => "Gengar Crest",
+	:desc => "Grants Misty Surge and a Fairy sub-type.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
+:INTELECREST => {
+	:ID => 873,
+	:name => "Inteleon Crest",
+	:desc => "Grants Acceleration.",
+	:price => 0,
+	:crest => true,
+	:noUseInBattle => true,
+	:noUse => true,
+},
 }
diff --git a/Pokemon Desolation/montext.rb b/Pokemon Desolation/montext.rb
index ef4557a..b775378 100644
--- a/Pokemon Desolation/montext.rb	
+++ b/Pokemon Desolation/montext.rb	
@@ -5882,7 +5882,7 @@ MONHASH = {
 			[50,:DESTINYBOND],
 			[55,:HEX],
 			[61,:NIGHTMARE]],
-		:compatiblemoves => [:ALLYSWITCH,:BIDE,:BODYSLAM,:BRICKBREAK,:CORROSIVEGAS,:COUNTER,:CURSE,:DARKPULSE,:DAZZLINGGLEAM,:DOUBLEEDGE,:DRAINPUNCH,:DREAMEATER,:DYNAMICPUNCH,:EMBARGO,:ENCORE,:ENERGYBALL,:EXPLOSION,:FIREPUNCH,:FLING,:FOCUSBLAST,:FOCUSPUNCH,:FOULPLAY,:GIGADRAIN,:GIGAIMPACT,:HEADBUTT,:HEX,:HYPERBEAM,:HYPNOSIS,:ICEPUNCH,:ICYWIND,:IMPRISON,:INFESTATION,:KNOCKOFF,:LASERFOCUS,:MEGADRAIN,:MEGAKICK,:MEGAPUNCH,:METRONOME,:MIMIC,:NASTYPLOT,:NIGHTMARE,:NIGHTSHADE,:OMINOUSWIND,:PAINSPLIT,:PAYBACK,:PHANTOMFORCE,:POISONJAB,:POLTERGEIST,:POWERUPPUNCH,:PSYCHIC,:PSYCHUP,:PSYWAVE,:RAGE,:RAINDANCE,:ROCKSMASH,:ROLEPLAY,:SCARYFACE,:SEISMICTOSS,:SELFDESTRUCT,:SHADOWBALL,:SHADOWCLAW,:SKILLSWAP,:SKITTERSMACK,:SKULLBASH,:SLUDGEBOMB,:SLUDGEWAVE,:SNATCH,:SPITE,:STRENGTH,:SUBMISSION,:SUCKERPUNCH,:SUNNYDAY,:TAKEDOWN,:TAUNT,:TELEKINESIS,:THIEF,:THUNDER,:THUNDERBOLT,:THUNDERPUNCH,:TORMENT,:TRICK,:TRICKROOM,:UPROAR,:VENOSHOCK,:WILLOWISP,:WONDERROOM,:ZAPCANNON,],
+		:compatiblemoves => [:ALLYSWITCH,:BIDE,:BODYSLAM,:BRICKBREAK,:CORROSIVEGAS,:COUNTER,:CURSE,:DARKPULSE,:DAZZLINGGLEAM,:DOUBLEEDGE,:DRAINPUNCH,:DREAMEATER,:DYNAMICPUNCH,:EMBARGO,:ENCORE,:ENERGYBALL,:EXPLOSION,:FIREPUNCH,:FLING,:FOCUSBLAST,:FOCUSPUNCH,:FOULPLAY,:GIGADRAIN,:GIGAIMPACT,:HEADBUTT,:HEX,:HYPERBEAM,:HYPNOSIS,:ICEPUNCH,:ICYWIND,:IMPRISON,:INFESTATION,:KNOCKOFF,:LASERFOCUS,:MEGADRAIN,:MEGAKICK,:MEGAPUNCH,:METRONOME,:MIMIC,:MOONGEISTBEAM,:NASTYPLOT,:NIGHTMARE,:NIGHTSHADE,:OMINOUSWIND,:PAINSPLIT,:PAYBACK,:PHANTOMFORCE,:POISONJAB,:POLTERGEIST,:POWERUPPUNCH,:PSYCHIC,:PSYCHUP,:PSYWAVE,:RAGE,:RAINDANCE,:ROCKSMASH,:ROLEPLAY,:SCARYFACE,:SEISMICTOSS,:SELFDESTRUCT,:SHADOWBALL,:SHADOWCLAW,:SKILLSWAP,:SKITTERSMACK,:SKULLBASH,:SLUDGEBOMB,:SLUDGEWAVE,:SNATCH,:SPITE,:STRENGTH,:SUBMISSION,:SUCKERPUNCH,:SUNNYDAY,:TAKEDOWN,:TAUNT,:TELEKINESIS,:THIEF,:THUNDER,:THUNDERBOLT,:THUNDERPUNCH,:TORMENT,:TRICK,:TRICKROOM,:UPROAR,:VENOSHOCK,:WILLOWISP,:WONDERROOM,:ZAPCANNON,],
 		:moveexceptions => [],
 		:Color => "Purple",
 		:Habitat => "Cave",
